<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ImageAddProgress</class>
 <widget class="QWidget" name="ImageAddProgress" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>117</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>261</width>
     <height>117</height>
    </size>
   </property>
   <property name="frameShape" >
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2" >
     <widget class="QProgressBar" name="progressBar" >
      <property name="value" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="imageLabel" >
      <property name="minimumSize" >
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="nameLabel" >
      <property name="text" >
       <string/>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
